### ISSUE TITLE
How to train it well?

### ISSUE THREAD

**Issue by mengxia1994** [GeorgeSeif/Semantic-Segmentation-Suite#219]
I use train.py to run about 300 epoch, other settings are also default(CamVid, FC-DenseNet56). But when i use the model to predict a image, its performance is very bad. Then i use one of the train image. It is much better but still far away from satisfied.
The val performance stop improving from almost 250 epoch, precision and recall won't surpass 0.9.  What should i do to obtain a model just like the main page show?

**Comment by mojoee** [comment]
i am also having issues training the Network with my own dataset at the moment. I have tried the usual things, as e.g. adjusting learning rate, optimizer, crop size, batch-size but it doesn't help
